types/testing-library__user-event/index.d.ts

```diff
@@ -1,18 +1,25 @@
 // Type definitions for @testing-library/user-event 4.1
 // Project: https://github.com/testing-library/user-event
 // Definitions by: Wu Haotian <https://github.com/whtsky>
+//                 Weyert de Boer <https://github.com/weyert>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export interface UserOpts {
-    allAtOnce?: boolean;
-    delay?: number;
+export interface UserOptions {
+  allAtOnce?: boolean;
+  delay?: number;
 }
 
+type TargetElement = Element | Window;
+
 declare const userEvent: {
-    click: (element: Element | Window) => void;
-    dblClick: (element: Element | Window) => void;
-    selectOptions: (element: Element | Window, values: string | string[]) => void;
-    type: (element: Element | Window, text: string, userOpts?: UserOpts) => Promise<void>;
+  click: (element: TargetElement) => void;
```

Comment:
Is this spacing correct? It keeps moving around.

Reply:
Good point, I am not sure if TypeDefinitely is reformatting or I am having some wacky prettier settings in VSCode 🤔